rhizome/host/lib/storage_volume.rb

```diff
@@ -196,6 +197,40 @@ def vhost_backend_create_service_file
     SERVICE
   end
 
+  def systemd_io_rate_limits
+    limits = {IOReadBandwidthMax: @max_read_mbytes_per_sec,
+              IOWriteBandwidthMax: @max_write_mbytes_per_sec}.compact
+    return "" if limits.empty?
+
+    dev = persistent_device_id(storage_dir)
+    limits.to_a
```

Comment:
```suggestion
    limits
```

`map` works for both arrays and hashes (actually, any enumerable).